tower_cli/models/base.py

```diff
@@ -484,7 +484,8 @@ def _pop_none(self, kwargs):
         click is unfortunately bad at the way it sends through unspecified
         defaults."""
         for key, value in copy(kwargs).items():
-            if value is None:
+            # options with multiple=True return a tuple
+            if value is None or value == ():
```

Comment:
I said I would prove more detail, so I'll do it in-line:

This is needed to more fully support the use of multiple=True in the case of extra_vars or other fields. Options that allow multiples always return as a tuple. If an input value is a tuple type and is empty, then we can be confident that click is telling us "the user has not specified any values".